src/uu/ls/src/ls.rs

```diff
@@ -3542,3 +3542,15 @@ fn calculate_padding_collection(
 
     padding_collections
 }
+
+fn escape_name(name: &OsStr, style: &QuotingStyle) -> String {
+    uucore::quoting_style::escape_name(name, style)
```

Comment:
I would `use uucore::quoting_style` and then call `quoting_style::escape_name` for best readability.

Reply:
Done.